### DISCUSSION TITLE
identify a real-life problem in financial sector and create a python application to solve this problem. • your program must contain any of the following or more: functions, arguments, generators, exceptions, inheritance, and a minimum of 5 variables. github<Your question or feedback about the GitHub Code Search and Code View products>

### DISCUSSION THREAD

**Discussion by a0799r** [community/community#120001] [Programming Help]
### Select Topic Area

Question

### Body

how can I go about the problem?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @a0799r !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Code Search and Navigation category is solely related to conversations around the [GitHub product Code Search and Navigation](https://github.com/search?q=&type=repositories). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

P.S. I always start development projects with [pseudo coding](https://www.geeksforgeeks.org/how-to-write-a-pseudo-code/). 
comment in a basic outline and fill in the steps as you go.   

As for real life problems in the financial sector... Holy cow thats a large pool full of mines! Remember the KISS method: Keep It Simple Stupid. Its tax season, maybe something about updating last years tax prep documentation with new tax law? - creating a change log?

**Comment by AIcoding-girl** [comment]
Using AI help, simplest 'problem' to solve seems **automated financial health check** using data like income, expenses, balance, assets, loans and investments. All the calculations are pretty straight forward and can be extended if needed. Good luck!
